File: Notes-Asbjørn/Timing/time2.py
import time

def usersleep():
	millis = int(round(time.clock() * 1000))
	t = 10000
	while True:
		if int(round(time.clock() * 1000)) - millis > t:
			break;

millis = int(round(time.clock() * 1000))
usersleep()
millisEnd = int(round(time.clock() * 1000)) - millis
print "time: %s" % millisEnd

File: Notes-Asbjørn/Timing/time2.cpp
#include <stdio.h>
#include <iostream>
#include <time.h>

int main( int argc, const char* argv[] )
{	
	clock_t t;
	t = clock();
	
    int goal = 10000000;
    
    while(true)
    {
    	if(clock() - t > goal)
    	{
    		break;
    	}
    }
    std::cout << "Time: " << (clock() - t)/1000 << " ms" << std::endl;
}
